### ISSUE TITLE
The error "missing field `reply_in_chat_id`"

### ISSUE THREAD

**Issue by micl2e2** [melix99/tdgrand#28]
Hi!

I was trying to use `tdlib` to do some simple tasks. For example, get latest 20 messages from each group. 

I added the following code to `example/get_me.rs` just after line `println!("Hi, I'm {}", me.first_name);` :

```rs
let chats = functions::get_chats(None, 100, client_id).await.unwrap();
if let tdlib::enums::Chats::Chats(chats) = chats {
    for chat_id in chats.chat_ids {
        dbg!(chat_id);
        if let Ok(info) = functions::get_chat_history(chat_id, 0, 0, 20, false, client_id).await
        {
            dbg!(info);
        }
    }
}
```

Then I got an error "missing field `reply_in_chat_id`", wonder if the code above is the correct way to use `tdlib-rs` API? 

Thanks!

**Comment by seeya** [comment]
Having the same issue too. I took a look at the place it threw the error `out/generated.rs:15386:45`. It seems like the response from Telegram doesn't include `reply_in_chat_id` which `struct Message` requires. 

The documentation from TDLib [here](https://core.telegram.org/tdlib/docs/classtd_1_1td__api_1_1message.html#ad1c0b461cd0ee40be6bf27af488f0db0) states that it should be 0 if it's not used. But strangely the key isn't there. 

```
If non-zero, the identifier of the message this message is replying to; can be the identifier of a deleted message.
```

If I edit the `generated.rs` file and remove the following it works ok. I don't think this is the correct fix, maybe setting it as an `Option` will be safer. What do you guys think? 
```rust
pub reply_in_chat_id: i64,
pub reply_to_message_id: i64,
```

Another way is to fork this project and modify `tdlib/tl/api.tl`. Removing the definitions of `reply_in_chat_id` and `reply_to_message_id` in the Message definition. I tried it and it works. https://github.com/seeya/tdlib-rs.git 


```rust
#[allow(clippy::too_many_arguments)]
    pub async fn send_message(chat_id: i64, message_thread_id: i64, reply_to_message_id: i64, options: Option<crate::types::MessageSendOptions>, input_message_content: crate::enums::InputMessageContent, client_id: i32) -> Result<crate::enums::Message, crate::types::Error> {
        let request = json!({
            "@type": "sendMessage",
            "chat_id": chat_id,
            "message_thread_id": message_thread_id,
            "reply_to_message_id": reply_to_message_id,
            "options": options,
            "input_message_content": input_message_content,
        });
        let response = send_request(client_id, request).await;
        if response["@type"] == "error" {
            return Err(serde_json::from_value(response).unwrap())
        }
        println!("{:#?}", response);
        Ok(serde_json::from_value(response).unwrap())
    }
```

**Comment by melix99** [comment]
> If I edit the generated.rs file and remove the following it works ok. I don't think this is the correct fix, maybe setting it as an Option will be safer. What do you guys think?

Well, this should be fixed in tdlib for a proper fix I'd say. I wonder if this is still an issue with latest tdlib. I just pushed a branch in https://github.com/paper-plane-developers/tdlib-rs/pull/29, can you try if it still reproduces with that? Otherwise we can manually edit the .tl file to make those properties optional until the bug in tdlib is fixed.

EDIT: I just noticed that the property in the new .tl has the "; may be null if none" string, which should be picked correctly by the generator, so I'm fairly sure it's fixed. I'll still wait to be sure though :)

**Comment by micl2e2** [comment]
> I just pushed a branch in #29, can you try if it still reproduces with that?

As far as I'm concerned, the originally posted example now works perfectly fine, thanks for the patch! @melix99

**Comment by melix99** [comment]
Nice, then this was fixed in tdlib already. Thanks for testing @micl2e2!

I merged #29 and released tdlib-rs 0.8.0, to I'll close this one.
